spec/controllers/events_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe EventsController, type: :controller do
+  # before(:all) {
+  #         @user = User.new(username: "testinguser", email: "test@testing.com")
+  #         log_in @user
+  #     }
+
+  it { should route(:get, root_path).to(action: :index) }
+  it { should route(:get, event_path(1)).to(action: :show, id: 1) }
+
+  describe 'GET #new' do
+    it { expect(response).to have_http_status(:success) }
+  end
+
+  describe 'GET #index' do
+    it { expect(response).to have_http_status(:success) }
+  end
+
+  # describe 'POST #create' do
```

Comment:
Please remove commented out code. 
As can be seen from [this screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
It is better to delete commented code. If you need to review a previous version of your code, use git history. 🙂